release_notes/unreleased.md

```diff
@@ -1 +1,5 @@
 **Unreleased**
+
+* [SOARHELP-2713] Authentication method selection is available in asset configuration. 
+    * User can choose between: default, ssl, basic, credssp, ntlm.  
+    * Default method means that while FIPS is enabled NTLM will be used, otherwise Basic HTTP will be applied
```

Comment:
This is the opposite of what you wrote in the README.

Reply:
Yeah, thanks for pointing it out, readme one is correct